src/HucaresServer.Storage/Helpers/MissingPlateHelper.cs

```diff
@@ -15,7 +15,6 @@ public MissingPlateHelper(IDbContextFactory dbContextFactory = null)
             _dbContextFactory = dbContextFactory ?? new DbContextFactory();
         }
 
-        /// <inheritdoc />
         public MissingLicensePlate InsertPlateRecord(string plateNumber, DateTime searchStartDatetime)
```

Comment:
This method has no validation for plateNumber and never throws FormatException...